pkg/arch_test.go

```diff
@@ -0,0 +1,13 @@
+package pkg
+
+import (
+	"testing"
+
+	"github.com/matthewmcnew/archtest"
+)
+
+func TestDependencies(t *testing.T) {
```

Comment:
does this impact test times in a significant way?

Reply:
It will only scan direct dependencies so the test time increase is manageable.

Reply:
Looks like the testing time, at least in actions, did not change